### DISCUSSION TITLE
Github for my school, document issue

### DISCUSSION THREAD

**Discussion by waldette2** [community/community#65180] [GitHub Education] [answered]
### Hi everyone,

Good morning, I have submitted the request to be a part of the Github Copilot program for my school. However, they responded saying that the document does not identify itself as a student document. I am a teacher, I am from Catalonia, Spain, and I don't know how to prove that I am working at a school. I have sent my teacher's ID, but they say it's not valid. What can I do?

Thanks

**Comment by Ket-0626** [comment]
It sounds like you're facing an issue with proving your affiliation as a teacher from a school in Catalonia, Spain, for the GitHub Copilot program. If your teacher's ID isn't being accepted as valid documentation, here are a few steps you could consider taking to resolve the situation:

1. **Contact GitHub Support**: Reach out to GitHub's support team or the GitHub Copilot support specifically. Explain your situation in detail and provide any additional information they might need. They might be able to guide you on the specific types of documentation they accept as proof of affiliation.

2. **School Letter or Official Document**: Try obtaining a letter from your school's administration or a relevant school official that confirms your employment as a teacher at the institution. This letter should be on official school letterhead, include your name, role, and contact information of the school's administration. It should also ideally be signed or stamped by a school authority.

3. **Verification Email**: If your school has an official email domain, you could try using that email to sign up for the GitHub Copilot program. This could potentially serve as verification of your affiliation as a teacher.

4. **Payroll or Employment Documentation**: Provide a recent payslip or any official employment documentation that clearly shows your name and your position as a teacher at the school.

5. **Contact Your School**: Reach out to the administration of your school and explain the situation to them. They might have encountered similar issues before or be able to provide additional documentation or assistance.

6. **Alternative Proof**: If GitHub continues to have issues with your documentation, inquire if there are alternative methods of verifying your affiliation, such as through a phone call or video call with a GitHub representative.

7. **Patience and Persistence**: Sometimes, these verification processes can be a bit complex. Be patient and persistent in your communication with GitHub support. They will likely want to help you resolve the issue and get you onboarded if your affiliation can be verified.

Remember that each platform may have specific requirements and procedures for verification. It's always a good idea to carefully review the verification requirements they provide and tailor your documentation accordingly.

**Reply by waldette2** [reply]
Thanks, If my new request isn't work with that new document, I'll try to make a letter sign it for my school leader. Thanks

**Comment by arafajs** [comment, marked as answer]
Try to submit any other educational document. in the end, you will pass for sure. 🎯

**Reply by waldette2** [reply]
Thanks all, I try to create a new request with another document, the official contract. I expect it works. I'll inform you of the results.
Thanks

**Comment by waldette2** [comment]
Another rejected request, and now i can't upload again my proof because it say i used it... And i don't know how to continue a rejected one.

I think i don't try more. I'll find it another ai for code with my stundents and this is all. I can't do more for demostrate that i'm working in this school.

Thanks anyway.

**Comment by waldette2** [comment]
Thanks all, I tried my next option and I'm verified now! :D

**Reply by kpripper** [reply]
What is the next option? What do you mean?

**Comment by waldette2** [comment]
Ah, another thing: How can I see if I have Copilot activated? I want to test it previously to teach my stundents...
